File: android/LocShare/app/src/main/java/com/autod/gis/locshare/model/Response.kt
package com.autod.gis.locshare.model

import com.google.gson.internal.LinkedTreeMap

class Response
{
    var data: Any? = null
    var message: String? = null
    var succeed: Boolean = false

    fun getLinkedTreeMapData():LinkedTreeMap<String,Any>
    {
        return  data as LinkedTreeMap<String, Any>
    }
    fun getLinkedTreeMapListData():List<LinkedTreeMap<String,Any>>
    {
        return  data as List<LinkedTreeMap<String, Any>>
    }
}

File: asp.net/LocShare/Dto/Response.cs
﻿using Newtonsoft.Json;

namespace LocShare.Dto
{
    public class Response<T>
    {
        [JsonProperty("data")]
        public T Data { get; set; }

        [JsonProperty("message")]
        public string Message { get; set; }

        [JsonProperty("succeed")]
        public bool Succeed { get; set; } = true;
    }
}